index.html

```diff
@@ -12,7 +12,7 @@
   <div id="blurBg">
     <header>
       <a href="#"><h2>John Doe</h2></a>
-      <div class="menuImg"><img src="./images/menu.png" class="menu" alt=""></div>
+      <div class="menuImg"><img src="./images/menu.png" class="menu" alt="Menu"></div>
```

Comment:
> |Figma| Yours|
> |----|----|
> | <img width="311" alt="im4" src="https://user-images.githubusercontent.com/77843050/204755763-aa4818ff-a0d3-4d45-aa84-b2f575080942.png"> | <img width="372" alt="lol" src="https://user-images.githubusercontent.com/77843050/204755863-66c2b1a9-5b24-4834-85c1-37b9159c6479.png"> |

- [ ] Let me ask, are you called `John Doe`? 😆, the answer is `no` I suppose, funny right? this is to say you should replace all the sample names in the app with yours since this portfolio belongs to you. Please remember to update it as we move on to the next phases of the project. 
- [ ] Seeing the text font in the 2 images, your `hey there` font and your `p`  are a bit different from the original design, please update the styling by using the correct `font, padding, margin, and line-height.` etc...
- [ ] There is a white space at the bottom of your app, please adjust the height to cover the whole page, please

Reply:
Did you check the font names also, after checking the look?